tests/testthat/test-rcs.R

```diff
@@ -5,17 +5,31 @@ vpts <- example_vpts
 vpi <- integrate_profile(example_vpts)
 
 test_that("rcs() returns error on incorrect parameters", {
-  expect_error(rcs("not_a_vp"))
-  expect_error(rcs(vp_list_mixed), "`x` must be list of `vp` objects.", fixed = TRUE)
+  expect_error(
+    rcs("not_a_vp"),
+    regexp = "no applicable method for 'rcs' applied to an object of class \"character\"",
+    fixed = TRUE)
+  expect_error(
+    rcs(vp_list_mixed),
+    "`x` must be list of `vp` objects.",
+    fixed = TRUE)
 })
-
+# TODO: IMPROVE THIS ERROR MESSAGING, NOT DESCRIPTIVE
 test_that("rcs()<- returns error on incorrect parameters", {
-  expect_error(rcs(vp) <- "not_numeric")
-  expect_error(rcs(vp) <- NULL)
+  expect_error(rcs(vp) <- "not_numeric",
+               regexp = "value is not a numeric or integer vector",
+               fixed = TRUE)
+  expect_error(rcs(vp) <- NULL,
+               regexp = "value is not a numeric or integer vector",
+               fixed = TRUE)
   expect_error(rcs(vp) <- c(2, 2))
   expect_error(rcs("not_a_vp") <- 5)
```

Comment:
These two expectations are succeeding, because they result in tests. However, the messages are confusing. 

I'm unsure of what the expectations actually are here, so I couldn't fix them. 

```R
> rcs(vp) <- c(2, 2)
#> Error: assert_that: length of assertion is not 1
> rcs("not_a_vp") <- 5
#> Error in rcs("not_a_vp") <- 5 : 
#>  target of assignment expands to non-language object
```

From the lines above it seems to me integer vectors should be allowed. The second expectation should probably be `no applicable method for 'rcs' applied to an object of ...` or an improvement upon the default.